spec/controllers/greetings_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspec'
+require 'spec_helper'
+require_relative '../../app/controllers/greetings_controller.rb'
```

Comment:
RSpec should be able to automatically require the appropriate controller file, thanks to the rails conventions and the magic the `rspec-rails` gem provides.